src/test/java/com/bravo/user/service/PaymentServiceTest.java

```diff
@@ -0,0 +1,87 @@
+package com.bravo.user.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import com.bravo.user.dao.model.Payment;
+import com.bravo.user.dao.model.mapper.ResourceMapper;
+import com.bravo.user.dao.repository.PaymentRepository;
+import com.bravo.user.model.dto.PaymentDto;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@SpringBootTest
+public class PaymentServiceTest {
+
+    @Autowired
+	private PaymentService paymentService;
+
+    @MockBean
+	private ResourceMapper resourceMapper;
+
+	@MockBean
+	private PaymentRepository  paymentRepository;
+
+	private List<PaymentDto> dtoPayments;
+
+    @BeforeEach
+	public void beforeEach() {
+        log.info("Initializing  payment list");
+		final List<Integer> userIds = IntStream.range(1, 10).boxed().toList();
+
+		final List<Payment> daoPayments = userIds.stream()
+				.map(id -> createPayment(Integer.toString(id))).collect(Collectors.toList());
```

Comment:
If you update the `createPayment` method to accept a `String` then you can use method reference instead of this Lambda. 

```
.map(this::createPayment)
```